docs/kms/README.md

```diff
@@ -26,7 +28,26 @@ export MINIO_SSE_VAULT_KEY_NAME=my-minio-key
 minio server ~/export
 ```
 
-### 4. Test your setup
+#### 2.2 Direct master key
+
+Alternatively you can specify a the master key directly using:
+```sh
+export MINIO_KMS_MASTER_KEY=MasterKey1:6368616e676520746869732070617373776f726420746f206120736563726574
+``` 
+
+**Important:** If you loose your master key you won't be able to decrypt the objects anymore. This means you lost
+all objects encrypted with the particular master key. Therefore you **MUST** make sure that you keep a backup of the
+master key at a secure location.  
+
+The master key must be 32 bytes long and provided as HEX value. The master key is referenced by a master key ID - in this example 
+`MasterKey1` - and is separated from the master-key by a `:`. So valid values for `MINIO_KMS_MASTER_KEY` must match the following
+pattern: `MASTER_KEY_ID`:`32_BYTE_HEX_VALUE`.
+
+**Notice:** The master key is accessible for anyone who has access to the Minio server. In general it is recommended to use
+an external KMS service (Vault). From a security point of view specifying a KMS master key directly is only useful if the storage
+backend is not at the same machine as the Minio server instance - for example Minio in gateway mode.  
```

Comment:
> for example Minio in gateway mode.  
>
since encryption is disabled for all gateways with the exception of S3 (once PR #6423 is in), we might want to give a different usecase for specifying KMS master key directly to avoid confusion.

Reply:
Well, that's the primary use case: User runs Minio Gateway pointing to S3 to use AWS as scalable storage provider while keeping object content private. (No trust in AWS required). Other use cases - like storage is on a network drive - is also possible but no clue how common.

Reply:
ok